server/vulnerabilities/oval/oval_platform.go

```diff
@@ -0,0 +1,41 @@
+package oval
+
+import (
+	"fmt"
+	"regexp"
+	"strings"
+	"time"
+)
+
+// Combines the host platform and os version into 'platform-major version' string,
+// ex: ('ubuntu', 'Ubuntu 20.4.0') => 'ubuntu-20'
```

Comment:
This line comment belongs to `NewPlatform`.

Reply:
Was actually trying to document what the type was all about (since is just a wrapper around `string`) but if the comments make more sense on top of `NewPlatform`, will move them there - thanks